code/day2/5_actel_custom_spatial.R

```diff
@@ -0,0 +1,89 @@
+## Actel Part 2 - Custom Spatial Networks:
+
+# Let's design a spatial.txt file for our detection data
+
+# Let's first visualize our study area, with a popup that tells us what 
+# project each deployment belongs to:
+
+# Designing a spatial.txt file -----
+
+library(mapview)
+library(spdplyr)
+library(leaflet)
+library(leafpop)
+
+
+## Exploration - Let's use mapview, since we're going to want to move around, 
+#   drill in and look at our stations
+
+
+# Get a list of spatial objects to plot from actel_spatial_sum:
+our_receivers <- as.data.frame(actel_spatial_sum) %>%    
+  dplyr::filter(Array %in% (actel_spatial_sum %>%   # only look at the arrays already in our spatial file
+                              distinct(Array))$Array)
+
+rcvr_spatial <- our_receivers %>%    
+  dplyr::select(Longitude, Latitude) %>%           # and get a SpatialPoints object to pass to mapview
+  sp::SpatialPoints(CRS('+proj=longlat'))
+# and plot it using mapview. The popupTable() function lets us customize our tooltip
+mapview(rcvr_spatial, popup = popupTable(our_receivers, 
+                                         zcol = c("Array",
+                                                  "Station.name")))  # and make a tooltip we can explore
+
+# Can we design a spatial.txt file that fits our study area using 'glatos_array' 
```

Comment:
need to change `glatos_array` to act variable

Reply:
5_.R is a WIP and is probably getting cut, mapview's current version is not working (!) and when i trield creating a spatial.txt file for ACT i hit another Actel error and i don't want to have to unravel that in the next 40 hours, and we have too much content for this timeline, so i'm thinking i leave it out.

Reply:
Committing the WIP so we don't lose the work, but will remove it from the lesson markdown.